src/themes/components/carousel.ts

```diff
@@ -96,6 +96,79 @@ const numbersPaginationButton = {
   },
 };
 
+const dotsPaginationButtonWrapper = {
```

Comment:
could you please check the gap between the dots? In the design it's 6px, on your branch deployment it's more

Reply:
Because of the clickable buttons